src/coreclr/src/pal/tests/palsuite/exception_handling/pal_sxs/test1/CMakeLists.txt

```diff
@@ -18,18 +18,18 @@ convert_to_absolute_path(DEF_SOURCES1 ${DEF_SOURCES1})
 set(EXPORTS_FILE1 ${CMAKE_CURRENT_BINARY_DIR}/dlltest1.exports)
 generate_exports_file(${DEF_SOURCES1} ${EXPORTS_FILE1})
 
-if(CLR_CMAKE_TARGET_LINUX OR CLR_CMAKE_TARGET_FREEBSD OR CLR_CMAKE_TARGET_NETBSD)
+if(CLR_CMAKE_TARGET_LINUX OR CLR_CMAKE_TARGET_FREEBSD OR CLR_CMAKE_TARGET_NETBSD OR LD_GNU)
```

Comment:
I think we can work out the remaining ld flavors and simplify such conditions based on linker in use; instead of listing each platform individually (as the enclosing code in these block is related to linker options).